### DISCUSSION TITLE
contributing to the physics section

### DISCUSSION THREAD

**Discussion by pluto-tofu** [TheAlgorithms/Python#8852] [General]
Can anyone please guide me on how to contribute to the physics section. Can I find and implement various topics by myself or is there a specific list of the topics needed ?

**Comment by CaedenPH** [comment]
> Can anyone please guide me on how to contribute to the physics section. Can I find and implement various topics by myself or is there a specific list of the topics needed ?

@pluto-tofu You can implement whatever algorithm you want, there are no lists that only allow select algorithms. As long as you read CONTRIBUTING.md, feel free to make a pull request implementing any algorithm to any directory

**Reply by pluto-tofu** [reply]
@CaedenPH Thank you for the help!

**Reply by CaedenPH** [reply]
> @CaedenPH Thank you for the help!

@pluto-tofu No problem, if thats everything mark this discussion as answered so people can filter correctly via unanswered discussions
